scripts/build-win.js

```diff
@@ -0,0 +1,30 @@
+const fs = require('fs');
+const tp = require('teen_process');
+const log = require('fancy-log');
+
+const APKS_DIR = './apks';
+const GRADLE_DIR = require('path').resolve('./');
+const APK_FILENAME = '/ApiDemos-debug.apk';
+
+async function gradleExecution() {
+  let stderr;
+  try {
+    ({stderr} = await tp.exec('cmd.exe', ['/c', 'gradlew.bat', 'clean', 'assembleDebug'], {cwd: GRADLE_DIR}, {shell: true}));
```

Comment:
I don't see a need to provide `cwd` option. Also, is it mandatory to have `shell` set to true?

Reply:
According to teen_process docs:
`If you're on Windows, you'll want to pass shell: true, because exec actually uses spawn under the hood, and is therefore subject to the issues noted about Windows + spawn in [the Node docs](https://nodejs.org/api/child_process.html).`

Reply:
Regarding the cwd option, U are are right, no need for that.

Reply:
I would say if it works without shell then consider dropping it. Also, I assume this last argument is ignored anyway. The original function only accepts 3 args.

Reply:
I dropped it